slices/node-acorn.yaml

```diff
@@ -0,0 +1,35 @@
+package: node-acorn
+
+slices:
+  libs:
+    essential:
+      - node-xtend_libs
+    contents:
+      # basically avoiding all the .map files (line tables) and unittest drivers
+      /usr/share/nodejs/acorn/**:
+      /usr/share/nodejs/acorn-walk/**:
+      /usr/share/nodejs/acorn-static-class-features/index.js:
+      /usr/share/nodejs/acorn-static-class-features/package.json:
+      /usr/share/nodejs/acorn-private-methods/index.js:
+      /usr/share/nodejs/acorn-private-methods/package.json:
+      /usr/share/nodejs/acorn-private-methods/dist/*.js:
+      /usr/share/nodejs/acorn-private-methods/dist/*.mjs:
+      /usr/share/nodejs/acorn-private-class-elements/**:
+      /usr/share/nodejs/acorn-numeric-separator/dist/*.js:
+      /usr/share/nodejs/acorn-numeric-separator/dist/*.mjs:
```

Comment:
nit: I **think** the `mjs` and `js` lines could be merged into just 1, like:

`/usr/share/nodejs/acorn-numeric-separator/dist/*.*js:`

but I haven't tested...

Reply:
I think I will just keep the pattern more specific. Since `*.*js` may also match some potentially unneeded files in the future.